allensdk/brain_observatory/ophys/project_constraints.py

```diff
@@ -0,0 +1,23 @@
+"""Collection of specific project metadata not readily available on LIMS.
```

Comment:
pretty sure you meant to call this `project_constants`, not `project_constraints`

Reply:
Oops.